bioblend/galaxy/tools/__init__.py

```diff
@@ -189,6 +189,41 @@ def show_tool(self, tool_id, io_details=False, link_details=False):
         params['link_details'] = link_details
         return self._get(id=tool_id, params=params)
 
+    def build(self, tool_id, inputs=None, tool_version=None, history_id=None):
+
+        """
+        Get updated tool parameters of a given tool.
```

Comment:
Could the docstring be extended a bit, to explain what this method should be used for? An example of the `inputs` dict and the output dict would also be nice.

Reply:
@simonbray I added the descriptions for the method. Also, included detail description, examples and a test for build method. Thanks for reviewing the PR and your valuable suggestions.

Reply:
@simonbray Please let me know if you have any other comments or suggestions thank you.